src/apps/schedule/crud/events.py

```diff
@@ -148,3 +178,10 @@ async def get_by_event_id(self, event_id: int) -> int:
         result = await self._execute(query)
         flow_id: int = result.scalars().one_or_none()
         return flow_id
+
+    async def delete_all_by_event_ids(self, event_ids: list[int]):
+        """Delete all flow events by event ids."""
+        query: Query = update(FlowEventsSchema)
+        query = query.where(FlowEventsSchema.event_id.in_(event_ids))
+        query = query.values(is_deleted=True)
+        await self._execute(query)
```

Comment:
What about naming like `delete_by_ids()`?

Reply:
Same as ActivitySchema, we are deleting by event_id, not id of FlowEvent schema. Wouldn't it be confusing?